src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -309,11 +313,35 @@ const AddOrEditExperienceForm = ({
           />
         </Grid>
       )}
-      <Grid item xs={12} align="end">
+      <Grid
+        item
+        xs={12}
+        style={{display: 'flex'}}
+        className={
+          onDelete ? classes.alignButtonsBetween : classes.alignButtonsEnd
+        }
+      >
+        {onDelete ? (
+          <Button
+            className={classes.delete}
+            variant="text"
+            onClick={() => setOpenDeleteDialog(true)}
+          >
+            Delete
+          </Button>
+        ) : null}
```

Comment:
If you put, like `<span />` or something similar here, instead of `null`, you can get the same effect but still get to use `space-between`

Reply:
Great idea. Done.